bluepysnap/edges.py

```diff
@@ -614,8 +620,10 @@ def get(self, edge_ids, properties):
 
     def properties(self, edge_ids, properties):
         """Doc is overridden below."""
-        Deprecate.warn("EdgePopulation.properties function will be deprecated in 1.0.0. Please use "
-                       "EdgePopulation.get instead.")
+        Deprecate.warn(
+            "EdgePopulation.properties function is deprecated and will be removed in 1.0.0. "
+            "Please use EdgePopulation.get instead."
+        )
```

Comment:
Regarding these, these are only style changes and probably should not be a part of this PR, right?

That being said, the more I've worked on snap lately, the more I have wanted to just (in the lack of a better word) blackify it and let `black` do all the heavy lifting regarding the styling. What do you think?

Reply:
This is not only a format change, but I wanted to fix the message: the message was saying that "the function will be deprecated in 1.0.0", but the function is already deprecated in the current version. In 1.0.0 it will be removed (if not, the message should be fixed in a different way).
Even though the changes are not related to python 3.9, I did it in this PR while updating to python 3.9 because I was fixing also the warnings caused by the new version of panda.

About black, I'm in with it, also considering that it's now considered not beta anymore. We can discuss it directly 👍

Reply:
oh, you're absolutely correct. I missed that as it was not inline-highlighted. my bad. 🖐️
Okay, let's keep it here, then. 

Yeah, if you ask me, black is the new... eh, black? 😅